sync/class.jetpack-sync-defaults.php

```diff
@@ -74,6 +74,7 @@ class Jetpack_Sync_Defaults {
 		'disabled_reblogs',
 		'jetpack_comment_likes_enabled',
 		'twitter_via',
+		'jetpack-memberships-connected-account-id',
```

Comment:
Since you've added a new option here, you'll want to update the array in the tests as well to avoid failures with the unit tests:
https://github.com/Automattic/jetpack/blob/e49502abdfae4340d06ca6cd4f1662b9590978c6/tests/php/sync/test_class.jetpack-sync-options.php#L203

Reply:
Done!